khbr/KH2/data/enemies.yaml

```diff
@@ -1364,6 +1364,12 @@ Barbossa:
   blacklist_destination:
   - Xigbar # You are put in boxed in space
   - Riku # Allegedly doesn't finish the fight if you kill him - Kayya
+  - Blizzard Lord # Illuminator doesn't have a fun time here, causing Barbossa to disappear forever - Kayya
+  - Volcano Lord # Illuminator doesn't have a fun time here, causing Barbossa to disappear forever - Kayya
+  - Vexen # Illuminator spawns out of bounds - Kayya
+  - Oogie Boogie # Illuminator spawns out of bounds/spawns in a very hard to see location - Kayya
```

Comment:
Instead of adding these OOB issues as specific blacklist_destination, a "needs_teleport" tag should be added

Reply:
Ah, maybe I should pay attention to tags. Should I add this tag to Barbossa instead of all these specific destinations?